docs/rst/tutorials/cloud_tutorial.rst

```diff
@@ -0,0 +1,168 @@
+.. _vulcanexus_cloud_tutorial:
+
+Vulcanexus Cloud tutorial
```

Comment:
Perhaps `tutorial` could be removed since it is part of the `Tutorials` section, and instead include the word `Kubernetes` so readers know better what is the content.

Reply:
NIT: `Vulcanexus Cloud and Kubernetes`? `with Kubernetes` sounds to me like Kubernetes is included in Vulcanexus.